orbit/cmd/desktop/desktop_windows.go

```diff
@@ -99,3 +100,52 @@ func watchSystemTheme(iconManager *iconManager) {
 		}()
 	}
 }
+
+// blockWaitForStopEvent waits for the named event kernel object to be signalled
+func blockWaitForStopEvent(channelId string) error {
+	if channelId == "" {
+		return errors.New("communication channel name should not be empty")
+	}
+
+	// converting go string to UTF16 windows compatible string
+	targetChannel := "Global\\comm-" + channelId
+	ev, err := windows.UTF16PtrFromString(targetChannel)
+	if err != nil {
+		return fmt.Errorf("there was a problem generating UTF16 string: %w", err)
+	}
+
+	log.Info().Msg("Comm channel was acquired ")
```

Comment:
What do you mean by "acquired"? (nothing happened yet, right?)

Reply:
good catch! I optimized the code and forgot to move this line. It should go below after the OpenEvent() call succeeds. By "channel acquired" here I mean that our process got a handle on the named event kernel object, and can now listen if this object gets signaled (indicating in our cause that fleet-desktop app should terminate). I added logs to indicate this.